src/main/webapp/WEB-INF/tags/instructor/studentList.tag

```diff
@@ -31,16 +31,21 @@
                     <th id="button_sortstudentname-${courseIndex}" class="button-sort-none" onclick="toggleSort(this,4)">
                         Student Name <span class="icon-sort unsorted"></span>
                     </th>
-                    <c:if test="${not fromCourseDetailsPage}">
-                        <th id="button_sortemail-${courseIndex}" class="button-sort-none" onclick="toggleSort(this,5)">
-                            Email <span class="icon-sort unsorted"></span>
-                        </th>
-                    </c:if>
-                    <c:if test="${fromCourseDetailsPage}">
-                        <th id="button_sortstudentstatus" class="button-sort-none" onclick="toggleSort(this,5)">
-                            Status <span class="icon-sort unsorted"></span>
-                        </th>
-                    </c:if>
+                    <c:choose>
+                        <c:when test="${fromCourseDetailsPage}">
+                            <th id="button_sortstudentstatus" class="button-sort-none" onclick="toggleSort(this,5)">
+                                Status <span class="icon-sort unsorted"></span>
+                            </th>
+                            <th id="button_sortemail-${courseIndex}" class="button-sort-none" onclick="toggleSort(this,6)">
```

Comment:
Not related to your changes, but I don't like how ```toggleSort``` takes in a magic number, very fragile, @damithc perhaps we should open an issue to look into how we can improve this function?